app/index.js

```diff
@@ -76,9 +75,9 @@ app.use(function (req, res, next) {
     next(new errors.NotFoundError({message: 'Page not found'}));
 });
 
-app.use(function (err, req, res, next) {
+app.use(function (err, req, res) {
```

Comment:
This doesn't work anymore.
See for example: https://github.com/expressjs/express/issues/2896

Reply:
Good call! Didn't know that!